test/GAP/gap_to_oscar.jl

```diff
@@ -111,3 +111,61 @@ end
     val = GAP.evalstr( "[ [ E(4), 2 ], [ 3, 4 ] ]" )
     @test_throws GAP.ConversionError GAP.gap_to_julia(fmpq_mat, val)
 end
+
+@testset "single cyclotomics" begin
+    F, z = CyclotomicField(1)
+    @test F(GAP.evalstr("2^64")) == F(2)^64
+
+    F, z = CyclotomicField(5)
+    @test F(GAP.evalstr("Sqrt(5)")) == -2*z^3 - 2*z^2 - 1
+
+    F, z = CyclotomicField(15)
+    @test F(GAP.evalstr("E(5)")) == z^3
+    @test F(GAP.evalstr("E(3)")) == z^5
```

Comment:
```suggestion
    @test F(GAP.Globals.E(5)) == z^3
    @test F(GAP.Globals.E(3)) == z^5
```

and so one elsewhere in this file